bin/test_submission

```diff
@@ -1,2 +1,21 @@
 #!/usr/bin/env python
-from rampwf import test_submission  # noqa
+import sys
+from rampwf.utils.testing import assert_submission  # noqa
+
+ramp_kit_dir = './'
+ramp_data_dir = './'
+submission_name = 'starting_kit'
+for arg in sys.argv[1:]:
```

Comment:
Addressed in #24: https://github.com/glemaitre/ramp-workflow/blob/2603bfcbb8f71e0aa1569c30065020e5a1cd6808/rampwf/utils/command_line.py